### ISSUE TITLE
[Meta] Milestones

### ISSUE THREAD

**Issue by jbzdarkid** [jacob404/promod-future#225]
So 4.5 came out recently, with a number of bugs. I'm well aware that most of these are mine; partially because I wrote most of the code in 4.5, but also because I was very busy. Busy, here, not in the sense of other work. I was busy working on other plugins. In other words: I didn't know what was going into 4.5, so I was working on anything / everything and what was _done_ was going into 4.5.

It would have made my life simpler, and my work more focused (and probably less buggy) if I had some sense of what was going to be in each milestone.

Now, I'm not asking that we declare everything that's going to be in an update now. I don't think that's reasonable; people don't really know what they want to work on before they do it. Instead, I'd propose that people set milestones when they begin work on a plugin. This means that there should always be _at least_ two milestones open, with one being "short term" and one being "long term". An example scenario:

I start work on a small plugin, and expect to finish it within a week. When I assign myself to the issue, I also set a 4.6 (short term) milestone. On the other hand, if the plugin is something large, I'd assign it to the 4.7 (long term) milestone.

Plugins which are not in active development should not get a milestone! This means that if something "should be" getting done, but nobody is working on it, it shouldn't be part of the next release. There's plenty of things that "should be" done, but aren't. We don't have enough time to fix everything right now, so it's worth our while to only focus on what we want to do / have time to do.

This should ensure that releases aren't rushed, in that people won't assign something to a release that we're planning to put out in a week if they're just getting started.

Relatedly, I'd like the "High Priority" tag to be used with a little more care. I see that tag as being jacob's (or anyone's, but probably his) way of saying something _should_ get done for the next release, without explicitly assigning anyone to it. Something like "we're changing how health bonus works" would go under this header, as it's _big_.

tl;dr Assign your own milestone when you start development. Don't assign milestones to inactive work. High Priority is only for things that _have to_ be done, not things we'd _like to_ be done.
